apps/web/src/pages/templates/components/VariantItemCard.tsx

```diff
@@ -157,10 +162,12 @@ export const VariantItemCard = ({
   const Icon = stepIcon[channel ?? ''];
   const variantsCount = ('variants' in variant ? variant.variants?.length : 0) ?? 0;
   const isRoot = nodeType === 'variantRoot';
+  const isSelected = (isRoot && stepUuid === variantUuid) || variant.uuid === variantUuid;
```

Comment:
the variant card will have a border around:
- when the current variant is a root step
- or when the current variant is the one the URL corresponds to